c2corg_ui/static/js/documentediting.js

```diff
@@ -374,6 +389,25 @@ app.DocumentEditingController.prototype.handleMapFeatureChange_ = function(
 
 
 /**
+ * It doesn't associate, it just pushes it into associations array.
+ * @private
+ */
+app.DocumentEditingController.prototype.pushDocToAssociations_ = function() {
```

Comment:
I mean that for interacting with the API it's recommended to use (or create if not already available) a function from the apiService instead of adding the function directly in your directive. Ask @gberaudo for precisions :)